packages/jest-jasmine2/src/index.js

```diff
@@ -15,13 +15,48 @@ import type {TestResult} from 'types/TestResult';
 import type Runtime from 'jest-runtime';
 
 import path from 'path';
-import fs from 'fs';
 import callsites from 'callsites';
+import {SourceMapConsumer} from 'source-map';
 import JasmineReporter from './reporter';
 import {install as jasmineAsyncInstall} from './jasmine_async';
 
 const JASMINE = require.resolve('./jasmine/jasmine_light.js');
 
+// Copied from https://github.com/rexxars/sourcemap-decorate-callsites/blob/5b9735a156964973a75dc62fd2c7f0c1975458e8/lib/index.js#L113-L158
```

Comment:
@rexxars thoughts on exporting `extendCallsite` from `sourcemap-decorate-callsites`?

I wanted to use it directly, but I don't want the searching for sourcemaps - I have it already

Reply:
Or just expose an API where I can pass ina callsite and a sourcemap, that would be even better (as long as it's sync) 🙂

Reply:
I'd welcome the change, but I'm absolutely swamped with work - be happy if anyone would submit a PR, though.